packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -216,8 +216,8 @@ MeetingUtil.declineMeeting = (meeting, reason) =>
     reason
   });
 
-MeetingUtil.isUnjoined = (locusInfo) =>
-  locusInfo.parsedLocus && locusInfo.parsedLocus.self && locusInfo.parsedLocus.self.state !== _JOINED_;
+MeetingUtil.isGuestUnjoined = (locusInfo) =>
```

Comment:
looks like a bad merge happened at some point, but this file has two: `MeetingUtil.leaveMeeting` functions...
here: https://github.com/webex/webex-js-sdk/blob/75516f149747c06a27dc55f5095a4e69a961194f/packages/node_modules/%40webex/plugin-meetings/src/meeting/util.js#L160
and 
here: https://github.com/webex/webex-js-sdk/blob/75516f149747c06a27dc55f5095a4e69a961194f/packages/node_modules/%40webex/plugin-meetings/src/meeting/util.js#L175
seems like a bug?

Reply:
Thanks rob looks like second one is latest and it overrides the first one . removed